packages/graphql-toolbox/src/modules/HelpDrawer/HelpDrawer.tsx

```diff
@@ -17,177 +17,44 @@
-    onClickClose,
-    setShowSubComponent,
-    schema,
-    screen,
-}: {
-    showSubComponent: boolean;
-    onClickClose: () => void;
-    setShowSubComponent: Dispatch<SetStateAction<boolean>>;
-    schema?: GraphQLSchema;
-    screen: Screen;
-}) => {
-    const [selectedTile, setSelectedTile] = useState<string>("");
-
-    const handleOnClickSchemaDocTile = () => {
-        setSelectedTile(EditorViewTiles.SCHEMA_DOCS.toString());
-        setShowSubComponent(true);
-        tracking.trackOpenSchemaDocs({ screen, action: true, origin: "help drawer" });
-    };
-    const handleOnClickKeybindingsTile = () => {
-        setSelectedTile(EditorViewTiles.KEYBINDINGS.toString());
-        setShowSubComponent(true);
-    };
-
-    return (
-        <Fragment>
-            {showSubComponent ? (
-                <Fragment>
-                    {selectedTile === EditorViewTiles.SCHEMA_DOCS.toString() ? (
-                        <DocExplorerComponent
-                            schema={schema}
-                            onClickClose={onClickClose}
-                            onClickBack={() => setShowSubComponent(false)}
-                        />
-                    ) : null}
-                    {selectedTile === EditorViewTiles.KEYBINDINGS.toString() ? (
-                        <Keybindings onClickClose={onClickClose} onClickBack={() => setShowSubComponent(false)} />
-                    ) : null}
-                </Fragment>
-            ) : (
-                <Fragment>
-                    <EditorScreenTiles
-                        onClickShowDocs={handleOnClickSchemaDocTile}
-                        onClickShowKeybindings={handleOnClickKeybindingsTile}
-                    />
-                    <Resources showSchemaView={false} />
-                </Fragment>
-            )}
-        </Fragment>
-    );
-};
-
 export const HelpDrawer = ({ onClickClose, schema }: Props) => {
     const screen = useContext(ScreenContext);
     const [showSubComponent, setShowSubComponent] = useState<boolean>(false);
 
+    const CannyFeedbackButton = ({ screen }: { screen: Screen }): JSX.Element => {
+        return (
+            <a
+                data-test-help-drawer-canny-button
+                className="flex justify-start items-center"
+                href="https://feedback.neo4j.com/graphql"
+                target="_blank"
+                rel="noreferrer"
+                onClick={() => tracking.trackHelpLearnFeatureLinks({ screen, actionLabel: "Send Feedback" })}
+            >
+                <ChatBubbleOvalLeftEllipsisIconOutline className="h-6 w-6 mr-2" />
+                <p className="p-0 m-0">Send feedback</p>
+            </a>
+        );
+    };
+
     return (
         <div className="p-6 w-full" data-test-help-drawer>
             {!showSubComponent ? (
                 <div className="pb-6 flex justify-between items-center" data-test-help-drawer-title>
-                    <Fragment>
+                    <>
```

Comment:
You should be able to remove this fragment as there is a `div` parent.

Reply:
430650b1e